tests/other/request.test.js

```diff
@@ -1370,3 +1377,79 @@ describe('Trips stats', () => {
       });
   });
 });
+
+describe('Book Accommodations', () => {
```

Comment:
Can you create a separate file accommodations test file? because this file is already long adding on it will make it even longer and difficult to debug in the future.

Reply:
Unfortunately moving it to a separate file raises issues in the CI